### PR TITLE
ref #218: upgrade jsTree component because of deprecated use of docum…

### PR DESCRIPTION
**Pull request by pixeljunkie**
…ent.registerElement which will be removed from chrome march 2019

| Q             | A
| ------------- | ---
| Branch        | 6.2.x
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | chameleon-system/chameleon-system#218
| License       | MIT

### CODE REVIEW

File: src/MediaManagerBundle/Resources/public/lib/jstree/3.3.2/jstree.js [hunk 1]
```diff
@@ -13,55 +13,37 @@
...
     /*!
-     * jsTree 3.3.2
-     * http://jstree.com/
-     *
-     * Copyright (c) 2014 Ivan Bozhanov (http://vakata.com)
-     *
-     * Licensed same as jquery - under the terms of the MIT License
-     *   http://www.opensource.org/licenses/mit-license.php
-     */
+ * jsTree 3.3.7
```
[the remark this answers is not in the document]
Comment (pixeljunkie) [reply src/MediaManagerBundle/Resources/public/lib/jstree/3.3.2/jstree.js]: Reversed this and added 3.3.7 parallel instead. Will remove the old version the 6.3 branch.

File: src/MediaManagerBundle/Bridge/Chameleon/BackendModule/MediaManagerBackendModule.php [hunk 2]
```diff
@@ -386,7 +386,7 @@ public function GetHtmlFooterIncludes()
     {
         $includes = parent::GetHtmlFooterIncludes();
         $includes[] = '<script src="'.TGlobal::GetStaticURL(
-                '/bundles/chameleonsystemmediamanager/lib/jstree/3.3.2/jstree.js'
+                '/bundles/chameleonsystemmediamanager/lib/jstree/3.3.7/jstree.js'
```
Comment (umulmrum) [inline src/MediaManagerBundle/Bridge/Chameleon/BackendModule/MediaManagerBackendModule.php]: In line 362 the version needs to be adjusted too, right?

Comment (pixeljunkie) [reply src/MediaManagerBundle/Bridge/Chameleon/BackendModule/MediaManagerBackendModule.php]: Did this already.

**Comment by umulmrum** [comment]
Thanks @pixeljunkie

**Pull request merged**